src/lsdb/loaders/hipscat/hipscat_loading_config.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class HipscatLoadingConfig:
```

Comment:
Why does this exist? What will be here eventually? Can you add that to the docstring?

Reply:
It's meant to contain whatever parameters are needed for loading a catalog. I figured there'd be quite a few of those and it'd be better to have an object to store them rather than passing around loads of individual variables. Originally I had a few in here, but then made some changes and realized they weren't necessary. I think as we develop more and use fsspec we'll probably need this so I kept it in.